docker-compose.yml

```diff
@@ -15,7 +15,9 @@ services:
      - "cassandra_data:/var/lib/cassandra"
 
   tilerator:
-    build: ./tilerator
+    build:
+      context: .
+      dockerfile: tilerator/Dockerfile
```

Comment:
This change requires to update `.travis.yml` to build the docker images in the expected contexts.